.github/workflows/tmp.yml

```diff
@@ -0,0 +1,27 @@
+name: CodeQL query
```

Comment:
This file is temporary only for testing. Successful run: https://github.com/dsp-testing/qc-run2/actions/runs/1290761996